sdk/resourcemanager/Azure.ResourceManager/src/BicepModelReaderWriterOptions.cs

```diff
@@ -10,7 +10,7 @@ namespace Azure.ResourceManager
     /// <summary>
     /// Provides the options for reading and writing Bicep.
     /// </summary>
-    [Experimental(nameof(BicepModelReaderWriterOptions))]
+    [Experimental("AZURE0001")]
```

Comment:
should we make this a const somewhere with name `BicepModelReaderWriterOptions`?